Sidenote using footnote syntax (#546)
executablebooks/sphinx-book-theme: docs/content-blocks.md

````
`````

(margin:sidenote)=
## Sidenotes and marginnotes

> This theme has support for [Tufte-stype margin / side notes](https://edwardtufte.github.io/tufte-css/), with a UX similar to [pandoc-sidenote](https://github.com/jez/pandoc-sidenote).

Reviewer: ```suggestion
This theme has support for [Tufte-style margin / side notes](https://edwardtufte.github.io/tufte-css/), with a UX similar to [pandoc-sidenote](https://github.com/jez/pandoc-sidenote).
```

> This theme has support for [Tufte-stype margin / side notes](https://edwardtufte.github.io/tufte-css/). The UX of which has been inspired by [pandoc-sidenote](https://github.com/jez/pandoc-sidenote)

Reviewer: ```suggestion
This theme has support for [Tufte-stype margin / side notes](https://edwardtufte.github.io/tufte-css/), with a UX similar to [pandoc-sidenote](https://github.com/jez/pandoc-sidenote).
```

Sidenotes are numbered, and behave like footnotes, except they live in the margin and don’t force the reader to jump their eye to the bottom of the page.
For example, here is a sidenote[^ex].
On narrow screens, sidenotes are hidden until a user clicks the number.
If you're on a mobile device, try clicking the sidenote number above.

[^ex]: Here's my sidenote text!

    On narrow screens, this text won't show up unless you click the superscript number!

Marginnotes are not numbered, but behave the same way as sidenotes.
On mobile devices a symbol will be displayed that will display the marginnote when clicked[^exmn].
For example, there's a marginnote in the previous sentence, and you should see a symbol show to display it on mobile screens.

[^exmn]: {-} This is a margin note. Notice there isn’t a number preceding the note.

> :::{seealso}

Reviewer: I noticed that sidenotes and marginnotes don't work if you put them inside of admonitions. They get placed _inside_ the admonition rather than in the margin. Is this a relatively easy fix? If not then we could add a note that says this isn't possible and try fixing it in a follow-up PR.
